requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 autoflake
 black
 reorder-python-imports
+vulture
```

Comment:
neat find - [a tool for finding dead code](https://github.com/jendrikseipp/vulture)

Reply:
also has configuration and can be setup with pre-commit https://github.com/jendrikseipp/vulture#configuration